.github/workflows/codespell.yml

```diff
@@ -0,0 +1,37 @@
+---
+name: Codespell
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  codespell:
+    name: Check for spelling errors
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: codespell-project/actions-codespell@master
+        with:
+          skip: opensc.conf.example.in
+          ignore_words_list:
```

Comment:
I would rather have this in separate file (ideally named in a way that local codespell can use it). The yml does not look like accepting comments well

Reply:
Sure. Where do I put the `codespell_ignore_words_file.txt` file?

Reply:
In `.github`?

Reply:
either in `.github` or in root directory. I would probably like it in root, unless there would be any strong opinions against this. Ideally, with a new section about the way how to run the codespell locally in `CONTRIBUTING.md`.

Reply:
I prefer having this file in root too, I was afraid you would find the root would get too crowded. If needed, I can make it a hidden file, for example **`codespell_ignore_words_file.txt`** → **`.codespell_ignore_words_file.txt`**.

Reply:
I have added a few words in `CONTRIBUTING.md`.